.github/workflows/code-formatting.yml

```diff
@@ -7,8 +7,8 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
     branches: [master, "release/*"]
 
 jobs:
-  imports-check-isort:
-    name: Check valid import formatting with isort
+  pre-commit-check:
```

Comment:
Let's separate pre-commit and isort...
if you think it is needed to add another job, but do not drop isort

Reply:
why? it makes sense to me to directly run pre-commit

and pre-commit outputs what failed/passed per hook